pkg/scheduler/internal/queue/scheduling_queue.go

```diff
@@ -705,9 +705,12 @@ func (npm *nominator) AddNominatedPod(pi *framework.PodInfo, nodeName string) {
 func (npm *nominator) NominatedPodsForNode(nodeName string) []*framework.PodInfo {
 	npm.RLock()
 	defer npm.RUnlock()
-	// TODO: we may need to return a copy of []*Pods to avoid modification
-	// on the caller side.
-	return npm.nominatedPods[nodeName]
+	// Caller can safely modify the return value
```

Comment:
Incorporate this in L703:

```
// NominatedPodsForNode returns a copy of pods that are nominated to run on the given node,
```

And here you can say:

```suggestion
	// Make a copy of the nominated Pods so the caller can mutate safely.
```

Reply:
done :)